cnf-certification-test/networking/suite.go

```diff
@@ -124,6 +128,22 @@ func testListenAndDeclared(env *provider.TestEnvironment) {
 	}
 }
 
+func testNodePort(env *provider.TestEnvironment) {
+	badNamespaces := []string{}
+	for _, put := range env.Pods {
+		ginkgo.By(fmt.Sprintf("Testing service account for pod %s (ns: %s)", put.Data.Name, put.Data.Namespace))
+		if put.Data.Spec.ServiceAccountName == "" {
```

Comment:
Pod's [Service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/ ) have nothing to do with [services](https://kubernetes.io/docs/concepts/services-networking/service/), they are completely different things. This TC should iterate through the CNF namespaces and check whether the services installed in those namespaces have the .spec.type == NodePort.